frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -25,83 +22,105 @@ let updatedClusterJSON;
+      await browser.wait(until.presenceOf(storageClusterRow(uid)));
+      const kebabMenu = storageClusterRow(uid).$('button[data-test-id="kebab-button"]');
 
-    updatedClusterJSON = JSON.parse(
-      execSync(`kubectl get -o json -n ${namespace} ${kind} ${name}`).toString(),
-    );
+      await click(kebabMenu);
+      await click(addCapacityLbl);
+      await click(addCapacityBtn);
 
-    updatedCnt = _.get(updatedClusterJSON, 'spec.storageDeviceSets[0].count', undefined);
-    const statusCol = storageClusterRow(uid).$('td:nth-child(4)');
+      updatedClusterJSON = JSON.parse(
+        execSync(`kubectl get -o json -n ${NAMESPACE} ${KIND} ${name}`).toString(),
+      );
 
-    // need to wait as cluster states fluctuates for some time. Waiting for 2 secs for the same
-    browser.sleep(2000);
+      updatedCnt = _.get(updatedClusterJSON, 'spec.storageDeviceSets[0].count', undefined);
+      const statusCol = storageClusterRow(uid).$('td:nth-child(4)');
 
-    await browser.wait(until.textToBePresentInElement(statusCol, 'Progressing'));
-    await browser.wait(
-      until.textToBePresentInElement(statusCol.$('span.co-icon-and-text span'), 'Ready'),
-    );
+      // need to wait as cluster states fluctuates for some time. Waiting for 2 secs for the same
+      browser.sleep(2000);
 
-    updatedPods = JSON.parse(execSync(`kubectl get pod -o json -n ${namespace}`).toString());
-  }, 150000);
+      await browser.wait(until.textToBePresentInElement(statusCol, 'Progressing'));
+      await browser.wait(
+        until.textToBePresentInElement(statusCol.$('span.co-icon-and-text span'), 'Ready'),
+      );
 
-  it('Newly added capacity should takes into effect at the storage level', () => {
-    // by default 2Tib capacity is being added
-    expect(updatedCnt - previousCnt).toEqual(1);
-  });
+      updatedPods = JSON.parse(execSync(`kubectl get pod -o json -n ${NAMESPACE}`).toString());
+    }, 150000);
 
-  it('New osd pods corresponding to the additional capacity should be in running state', () => {
-    const isPodPresent = (podName) =>
-      previousPods.items.find((pod) => pod.metadata.name === podName) !== undefined;
+    it('Newly added capacity should takes into effect at the storage level', () => {
+      // by default 2Tib capacity is being added
+      expect(updatedCnt - previousCnt).toEqual(1);
+    });
 
-    const newPods = [];
-    updatedPods.items.forEach((pod) => {
-      if (!isPodPresent(pod.metadata.name) && pod.metadata.name.startsWith('rook-ceph-osd')) {
-        newPods.push(pod.metadata.name);
-      }
+    it('New osd pods corresponding to the additional capacity should be in running state', () => {
+      const isPodPresent = (podName) =>
+        previousPods.items.find((pod) => pod.metadata.name === podName) !== undefined;
+
+      const newPods = [];
+      updatedPods.items.forEach((pod) => {
+        if (!isPodPresent(pod.metadata.name) && pod.metadata.name.startsWith('rook-ceph-osd')) {
```

Comment:
use name pattern enum defnd in the consts folder for `rook-ceph-osd`

Reply:
Already done in my PR.